e2e/fragments/servedDocument.js

```diff
@@ -6,7 +6,11 @@ module.exports = {
     for (const fileType of documents) {
       await within(fileType, async () => {
         I.click('Add new');
-        await I.attachFile(fileType + '_0_document', file);
```

Comment:
Do we not need to wait here the same way we did here: https://github.com/hmcts/civil-damages-ccd-definition/pull/61/files#diff-16d556d81373efc2037457b8069f40ac0ecb4a9c658a632ff781cd26466d0a82L18